src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Can load a native assembly.Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to find an entry point named 'TestEntry' in shared library 'nativedll'."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 78
78:         [TestNativeClass2]::LoadNative() | Should -Not -BeNullOrEmpty

```